src/Chirp.Web/wwwroot/js/script.js

```diff
@@ -29,6 +29,27 @@ $(document).ready(function() {
                         thisForm.removeClass('followForm unfollowForm').addClass(newClass + 'Form');
                         thisForm.attr('action', '/?handler=' + newHandler);
                     });
+
+                    // notification
+                    // Determine action for the notification
+                    let action = button.hasClass('follow') ? "You have unfollowed" : "You have followed";
+
+                    // Set the action message and author name
+                    $('#action-message').text(action);
+                    $('#author').text(author);
+                    $('#notification').show();
+
+                    // Show/hide the follow message based on the action
+                    if(action === "You have followed") {
```

Comment:
I'm confused as to why there's not also a state such as: 
```cs
if (action === "You have unfollowed")
```

Does this mean the message isn't shown if a user unfollows someone?

Reply:
No worries. The unfollow text does show in the notification for the unfollow button, just on the else statement for the people unfollowing. The specific text is then hidden/shown based on which notification being shown.

<p id="follow-message">View their cheeps on your own timeline!</p>

The above message is only needed in the follow statement, not the unfollow statement which just says "you have followed {author}".

It can be a bit confusing when you switch classes and ID's back and forth, but here's the notification where things are changed.

<!-- outer div notification -->
<div id="notification" class="flw-notifc slide-in-blurred-bottom">
    <!--svg icon-->
    <div class="flex-it">
        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-fill" viewBox="0 0 16 16">
            <path d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2m.995-14.901a1 1 0 1 0-1.99 0A5.002 5.002 0 0 0 3 6c0 1.098-.5 6-2 7h14c-1.5-1-2-5.902-2-7 0-2.42-1.72-4.44-4.005-4.901z"/>
        </svg>
        <div>
            <!-- messages from ajax -->
            <span id="action-message"></span>
            <span id="author"></span>
            <p id="follow-message">View their cheeps on your own timeline!</p>
        </div>
    </div>
</div>

This is the html button with the follow class seen in action:

![image](https://github.com/ITU-BDSA23-GROUP10/Chirp/assets/5868629/b410dd18-4ca0-485e-b5a5-d037c2accbcb)

The class changes to unfollow once the button was clicked (author was followed), and the message can now change to "you have unfollowed {author}" if clicked again, since it doesn't have the follow class anymore.

![image](https://github.com/ITU-BDSA23-GROUP10/Chirp/assets/5868629/9ebfae0f-70f4-48b2-903a-d952c38d9459)

This is from the script showing what it partly does:

let action = button.hasClass('follow') ? "You have unfollowed" : "You have followed";

// etc.. same principle where it changes the text or what to be shown in the specific IDs
$('#action-message').text(action);
$('#author').text(author);
$('#notification').show();

I guess it could be more specified of course, but we're not using the follow/unfollow class anywhere else, and that's why i didn't do a if (action === "You have unfollowed").